WssElixir/app/src/main/java/com/example/wsselixir/view/HomeActivity.kt

```diff
@@ -0,0 +1,103 @@
+package com.example.wsselixir.view
+
+import android.app.Dialog
+import android.content.Intent
+import android.os.Bundle
+import android.widget.ArrayAdapter
+import android.widget.Spinner
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.example.wsselixir.R
+import com.example.wsselixir.adapter.FollowersAdapter
+import com.example.wsselixir.data.FollowerInformation
+import com.example.wsselixir.databinding.ActivityHomeBinding
+import com.example.wsselixir.databinding.DialogFollowerInfoBinding
+
+class HomeActivity : AppCompatActivity() {
+    private lateinit var mbtiSpinner: Spinner
+    private lateinit var binding: ActivityHomeBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        myMBTISpinner()
+        infoRegistration()
+        initFollowerRecyclerView()
+    }
+
+
+    private fun infoRegistration() {
+        val btnRegistration = binding.btnHomeRegistration
+
+        btnRegistration.setOnClickListener {
```

Comment:
```suggestion
        binding.btnHomeRegistration.setOnClickListener {
```
뷰바인딩을 사용하니 요렇게 바로 사용해도 좋을 듯합니다 ~~

Reply:
확실히 코드가 줄어드네용!! 감사합니다!! 적용해볼게요~